.travis.yml

```diff
@@ -0,0 +1,13 @@
+  - secure: U0t+ZY1Y7+g26nDRl1TPfSJ1UDALi0en93Q0Sqmj3tNP2KKDN7GshWA1T+XAtv2knmgTbfeWyFz2tfidIu2Mi/0MutBKhGGfcPpAiuhwlX+e1XTW/AhTnYpcbY+K+zKvMuuriQTe4q2WOq1MFji8fWDFV9cTC2WfbYJKwmowuiNr/V48r37QFEckOE03L/hFAskuuAdOp/vFI5AM2Q6quvDcO4aToTBy0DPC4aYlLLaqerWY5/v+/rtUK1hHB0wKH00SMr6ZMAxxOzpVYmd5Vwvhu3GCN84AtHd5AYjKYK+1bjQ60wN8bYBId78L6C9AcJpyGu9XJWP8iakzrYMbMXoI0ZcQR+mmi8hv7mUc7uHv0V2pdXmc0682eJxEyX8LSViPwaHJIAFDB4hrfhHPqrbbdfGicCxnvK9UbRF3Z4AbqB8egK1PCA3yLnYR0wAVwounJNoCCQ+MHbJDXf8Fh2FlPlGg2jCDiFakkaBPr/AkNLUwSU03kmuHWaJjNEfPAr5FwF7W9MBoofFT4k3C2vCcthrSFPClF1lxU+BTzNZQS39yvMf5esD75+gavqkA5GTr5kIgELpDd2d0pdI3HN8cWJDgDkZNWCOuVzN+xqhB/PA3h3V3dfgpFiH69jeeWknlMn9AB4sn2l5PiPebvo7kR6xgpuV5NMPcxQihD04=
+  - secure: XD/DvNF0dtvJaNfN1OgMveVb+HAl0zW0laZL62zyQR/PZ7O+KwnAzPDYX9LQcgtukMmOyObjOOfergWJA8g4IVOpwcjY8yMINuLJQRgOGasp0rJt88unLFL78k5JaOmr2rEFkKTHV7kV/j+rfTM9ffSrgx1aQg+jnM36fj0J2n8o71jOR0asp5p+FReTqyiMxHBBKvwUpWTr5XbcGi5tX5ERnJqTH+8gH7DZbSHpG3hWSJ9xtDpD4NwF0LuwNUqQJhTnA/NaNL+qRuFpT3RLjmbiqqzggsM9Dk2r1FbmZ5O+D1KzDWs0LKDh3ZRvJ6nS1hpPlP+O4jbEX7TagO2bUZkc9oSfcK1I3/AO41iTF/sMZII+UnBxxCKsYxApqAzlPX0lZfO9AovZLWAFyvJyBItmaK4UUdYgH3Db1lgP4MzZuSHhnk1r5snseJ21XyN5jiOAVPv8fAVxi55haLkU8gTxIqUZ92Po+uZ4N9H2RIKgf1nQOceZeGmqg3DRk+ozbaUTdUaSH51VSYJUwaaOR/YUd2CmpLhUrUOFv4IjmFh2dNFeBoPhB5KhBHE0BoNJVJVdRSBepNid3iVAvwZSkYmQRHnPc+usjUjbjhsKO+Tc4cScOd8tLqTGo8+/aFroqNXteijrafl+p86ziL4QXyxnm8v/34LmZrJUilkBf/0=
+  - secure: Dlb4ITopzpn5vghVRlha4S49wOInlizyyjkt1wfwuZVPKSvMJsGX+8daKst7QqZfFW5lwc0rxaFUbNI2HcnOFKxzLjuFQ6Zn2rvqWlm0OcbtVs+1qPzkbz5X3o2TlLcDID0PnGAuWi+uYPDyb9OUIl/bzb0VtEubnBPy6ais0hPnR79sjjsaUbc67F+lt/aoGlNdxR8XAxo16ILt3TESlouEqj4NAH+rFw/s5JYsc9T+mYlAcCWXKJgBpShzjoEjOozGk+bCcwuEWnimbiW6g1mCCufgsgnUWNYTrA7ITBJknNCHYO3LrOE/x5hQri44m/gVd8y6MPFQzwkSa9Hqh8AO2hUmrxIhMsFHl2JysRvtcxLR8Nz9jGzQFZrVNGer/hEgxLYtx3AWq0zFpxI2WJLnteEzewT3xlBrBZhBZGWn6F9Q385xLQgmzpKXiAdT1XEUEPQ0vYErcSPFHF756EwoYU0NOgmWBnXmj48JiMSikwYUvXx/HWsns5sRY8mHGvc1dQBNOCOEINhc+f0jbT/x4TyvZ2uXodP3Dh833p4bXktQlUFgSZ4MV1AE7AN99y5nuM98abT8+g/fLqS/WCc36mpnvjjuR2X/vMvVJVEObL2YyNCHgNRYgEUJrS9+x7YKSGmeCvCN0kLY20p5SHyRCR0qpjWNCXsNXWcvin4=
+  - secure: TU4u7+YirlTCDAR5nBj0g6oRM1n+/j6kIS+9OwhEbwxZsnWJJi7cEWIQ3RdTczYC/eY2GwzYXOIUC8jDCX3NvkIEA00ev0lFZXqDX8qkqGD+ey1mqERTXs4LKpu2QP5ysxXWob3/+Jyw7l58l/Gxj6HFtikrgyGcORTjWr25cw9Es9y7F2+nyZDT5gA41OKOiTVb6WJYpVFKsHrflJRhqgXHqxtunygALo3nZn+p/RIx4wdofSC6H9KF8q8HQ4bFnqh3eUz2TtKNu8PxF3kJkG3vdTorIkN6nFjxj9HWNa1oMRqhrbYEDMiBPtcmHwm2KbEwo/DA5APx7SeHRBgUp6ZqCHq4EBCJP8JDsRt3EROXk4EQAVnRhVjLhZVJA8fgiJrDTPAGGpEeVFf1wsSveog+3myw1376JqYOC+u7gIiLI/fQP/0J5JMe4Dermk9J4aBEqnHTzvQPJq2m/OxijIDY7iIN9fOsvAawEBpGYPd0L9nO3iNymd6S6PJdmGqS+2yZ+c2EX2tZUOvkTz2x+t7jYsdB0pAGJhGF1/d7D5gdFc6W/1/ZghK6fKtDBKV+cPv5NCBxVpPD8X2E/qb4dUuaLpDqTWC8trw39hT5V9ezJU4YObTvz4Bnly2k0598O0IE63kuxT9ZI3W/JlZF2yOELArbh25sxQOnS3Taan4=
+script:
+- docker build --target test --tag my-test-image .
+- docker run --env-file env_test my-test-image test_client.py
+- docker run --env-file env_test my-test-image test_items.py
```

Comment:
Why did you rename `.env.test` to `env_test` and remove some of it?

You should restore the full `.env.test` file and get rid of env_test.

`poetry run pytest` still works locally but is not robust at the moment:
Your `test_client.py` is trying to load .env.test but not finding anything. And then Flask is automatically loading the real .env file. So the integration test is running with real credentials. If it made a request that wasn't mocked, if you were in the middle of writing a new test for example, then it would be making real, authorised requests to Trello. And any test would fail if it tried to assert that the app is making requests with `key=123` or with board id 123123.

And if you have the .env.test file, which the test loads with `dotenv`, you can get rid of `--env-file env_test` from these commands.